assets/js/blocks/cart-checkout-shared/icon.js

```diff
@@ -0,0 +1,24 @@
+const expressIcon = (
+	<svg
+		xmlns="http://www.w3.org/2000/svg"
+		width="24"
+		height="24"
+		fill="currentColor"
+		viewBox="0 0 24 24"
+	>
+		<path
+			stroke="#1E1E1E"
+			strokeLinejoin="round"
+			strokeWidth="1.5"
+			d="M18.25 12a6.25 6.25 0 1 1-12.5 0 6.25 6.25 0 0 1 12.5 0Z"
+		/>
+		<path fill="#1E1E1E" d="M10 3h4v3h-4z" />
```

Comment:
Just wondering if it's possible to utilize CSS variables here?

Reply:
Thanks for the suggestion. It's an SVG inside a JS file though so that won't work here.